docs/allowedwords.md

```diff
@@ -1,6 +1,6 @@
 ## How to add allowed word
 
-All the allowed words are located in an array in [src/config.ts](src/config.ts) inside the object ` ALEX ` as shown below. 
+All the allowed words are located in an array in [src/config.ts](https://github.com/EddieJaoudeCommunity/EddieBot/blob/develop/src/config.ts) inside the object ` ALEX ` as shown below. 
```

Comment:
I think we should change this so it would work on any branch/fork. What do you think?
```suggestion
All the allowed words are located in an array in [src/config.ts](../src/config.ts) inside the object ` ALEX ` as shown below. 
```
see: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files

Reply:
> I think we should change this so it would work on any branch/fork. What do you think?
> 
> see: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes#relative-links-and-image-paths-in-readme-files


Thank you so much for reviewing. Yes, you are right and now I've done the required changes so that it would work on any branch or fork.

Reply:
Yup, I thought about it, but in case there are further changes in the repositories then these relative links would have to be updated again. 
Would like to know the inputs of others regarding this.